src/Customizer.php

```diff
@@ -120,6 +125,19 @@ public function register() {
 			'section'  => 'sticky_menu_section',
 		) );
 
+		$this->wp_customize->add_control( 'sticky_menu_layout', array(
+			'type'     => 'select',
+			'priority' => 10,
+			'label'    => esc_html__( 'Sticky menu layout', 'pt-sticky-menu' ),
```

Comment:
@krumpak I would update the label to "Sticky menu visibility" and remove the description (it's not needed).